### PR TITLE
[INFRA-2545] Possible fix

### PR DESCRIPTION
**Pull request by slide**
Only add the originating user if it is not an org.

### CODE REVIEW

**Comment by slide** [comment]
@timja since you have looked at this source recently, can you take a look?

File: src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java [hunk 1]
```diff
@@ -948,15 +951,37 @@ boolean forkGitHub(Channel channel, User sender, String owner, String repo, Stri
 
             // GitHub adds a lot of teams to this repo by default, which we don't want
             Set<GHTeam> legacyTeams = r.getTeams();
+            GHTeam newTeam = null;
 
             try {
-                getOrCreateRepoLocalTeam(org, r, maintainers.isEmpty() ? singletonList(user.getName()) : maintainers);
+                // add the repo owner if there are no users specified and the repo owner is a user (not an org)
+                if(maintainers.isEmpty() && user.getType().equalsIgnoreCase("user")) {
```
Comment (timja) [inline src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java:958]: Don’t you need to clear the list if it’s not a user? I assume it’s still populated from before?

Comment (slide) [reply src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java:958]: The list is a non mutable list if it is empty at this point, so it requires being reallocated as a mutable list.

Comment (timja) [reply src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java:958]: i misread the code it's fine

**Pull request merged**